python/generate-dockerfile.sh

```diff
@@ -48,9 +50,10 @@ set_pythonpath() {
 		fi
 
 		if [ $1 != "2.7" ]; then
-			echo "ENV PYTHONPATH $PYTHON3_PATH" >> $tmp_path/Dockerfile
+			PPATH="PYTHON3_PATH_$4"
+			echo "ENV PYTHONPATH $(eval echo \$$PPATH):\$PYTHONPATH" >> $tmp_path/Dockerfile
```

Comment:
Ah, I get it, in that case you can use `echo "ENV PYTHONPATH ${!PPATH}:\$PYTHONPATH" >> $tmp_path/Dockerfile`